translate/convert/test_php2po.py

```diff
@@ -58,6 +60,34 @@ def test_convertphp(self):
         assert pounit.source == "entry"
         assert pounit.target == ""
 
+    def test_convertphptemplate(self):
+        """checks that the convertphp function is working with template"""
+        phpsource = """$_LANG['simple'] = 'entry';"""
+        phptemplate = '''$_LANG['simple'] = 'source';'''
+        posource = self.convertphp(phpsource, phptemplate)
+        pofile = po.pofile(wStringIO.StringIO(posource))
+        pounit = self.singleelement(pofile)
+        assert pounit.source == "source"
+        assert pounit.target == "entry"
+
+    def test_convertphpmissing(self):
+        """checks that the convertphp function is working with missing key"""
+        phpsource = """$_LANG['simple'] = 'entry';"""
+        phptemplate = '''$_LANG['missing'] = 'source';'''
+        posource = self.convertphp(phpsource, phptemplate)
+        pofile = po.pofile(wStringIO.StringIO(posource))
+        pounit = self.singleelement(pofile)
+        assert pounit.source == "source"
+        assert pounit.target == ""
```

Comment:
I would also assert the number of units in `pofile`.

Reply:
singleelement already does that

Reply:
That's right. Sorry for the noise.